ResearchKit/Common/ORKOrderedTask.m

```diff
@@ -1239,179 +1240,198 @@ + (ORKOrderedTask *)tremorTestTaskWithIdentifier:(NSString *)identifier
         }
     }
     
-    if (! (options & ORKPredefinedTaskOptionExcludeInstructions)) {
-        ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:@"instruction2"];
-        step.title = ORKLocalizedString(@"TREMOR_TEST_ACTIVE_STEP_IN_LAP_INTRO", nil);
-        step.text = ORKLocalizedString(@"TREMOR_TEST_ACTIVE_STEP_INTRO_TEXT", nil);
-        step.image = [UIImage imageNamed:@"tremortest3a" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
-        step.auxiliaryImage = [UIImage imageNamed:@"tremortest3b" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
-        step.shouldTintImages = YES;
-        
-        ORKStepArrayAddStep(steps, step);
-    }
-    
-    {
-        ORKCountdownStep *step = [[ORKCountdownStep alloc] initWithIdentifier:@"countdown1"];
-        
-        ORKStepArrayAddStep(steps, step);
-    }
-    
-    {
-        NSString *titleFormat = ORKLocalizedString(@"TREMOR_TEST_ACTIVE_STEP_IN_LAP_INSTRUCTION_%ld", nil);
-        ORKActiveStep *step = [[ORKActiveStep alloc] initWithIdentifier:ORKTremorTestInLapStepIdentifier];
-        step.recorderConfigurations = @[[[ORKAccelerometerRecorderConfiguration alloc] initWithIdentifier:@"ac1_acc" frequency:100.0], [[ORKDeviceMotionRecorderConfiguration alloc] initWithIdentifier:@"ac1_motion" frequency:100.0]];
-        step.title = [NSString stringWithFormat:titleFormat, (long)activeStepDuration];
-        step.spokenInstruction = step.title;
-        step.finishedSpokenInstruction = stepFinishedInstruction;
-        step.stepDuration = activeStepDuration;
-        step.shouldPlaySoundOnStart = YES;
-        step.shouldVibrateOnStart = YES;
-        step.shouldPlaySoundOnFinish = YES;
-        step.shouldVibrateOnFinish = YES;
-        step.shouldContinueOnFinish = NO;
-        step.shouldStartTimerAutomatically = YES;
-        
-        ORKStepArrayAddStep(steps, step);
-    }
-    
-    if (! (options & ORKPredefinedTaskOptionExcludeInstructions)) {
-        ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:@"instruction4"];
-        step.title = ORKLocalizedString(@"TREMOR_TEST_ACTIVE_STEP_EXTEND_ARM_INTRO", nil);
-        step.text = ORKLocalizedString(@"TREMOR_TEST_ACTIVE_STEP_INTRO_TEXT", nil);
-        step.image = [UIImage imageNamed:@"tremortest4a" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
-        step.auxiliaryImage = [UIImage imageNamed:@"tremortest4b" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
-        step.shouldTintImages = YES;
-        
-        ORKStepArrayAddStep(steps, step);
-    }
-    
-    {
-        ORKCountdownStep *step = [[ORKCountdownStep alloc] initWithIdentifier:@"countdown2"];
+    if (! (activeTaskOptions & ORKTremorActiveTaskOptionExcludeHandInLap)) {
```

Comment:
Nit: I prefer the explicit `if ((a & b) == 0) then do` or else a macro 

````
#define ORKFLAG_ON(flag, bitMask) ((flag & bitMask) == bitMask)
````

b/c I find it's a little easier for junior devs to grok.